README.md

```diff
@@ -3,73 +3,91 @@
 ![ci](https://github.com/probabl-ai/skore/actions/workflows/ci.yml/badge.svg?event=push)
 ![python](https://img.shields.io/badge/python-3.11%20|%203.12-blue?style=flat&logo=python)
 
-`skore` allows data scientists to create tracking and visualization from their Python code:
-1. Users can store objects of different types: python lists and dictionaries, `numpy` arrays, `scikit-learn` fitted models, `matplotlib`, `altair`, and `plotly` figures, etc. Storing some values over time allows one to perform **tracking** and also to **visualize** them:
-2. They can visualize these stored objects on a dashboard. The dashboard is user-friendly: objects can easily be organized.
-3. This dashboard can be exported into a HTML file.
+With `skore`, data scientists can:
+1. Store objects of different types from their Python code: python lists, `scikit-learn` fitted pipelines, `plotly` figures, and more.
+2. **Track** and  **visualize** these stored objects on a dashboard.
+3. Export the dashboard to a HTML file.
 
-These are only the first features of `skore`'s roadmap.
-`skore` is a work in progress and, on the long run, it aims to be an all-inclusive library for data scientists.
+These are only the first features: `skore` is a work in progress and aims to be an end-to-end library for data scientists.
 Stay tuned!
 
-<p align="center">
-    <img width="100%" src="https://github.com/sylvaincom/sylvaincom.github.io/blob/master/files/probabl/skore/2024_10_08_skore_demo.gif"/>
-</p>
+![GIF: short demo of `skore`](https://raw.githubusercontent.com/sylvaincom/sylvaincom.github.io/master/files/probabl/skore/2024_10_14_skore_demo.gif)
 
 ## ⚙️ Installation
 
-You can install `skore` by using `pip`:
+First of all, we recommended using a [virtual environment (venv)](https://docs.python.org/3/tutorial/venv.html). You need `python>=3.9`.
+
+Then, you can install `skore` by using `pip`:
 ```bash
 pip install -U skore
 ```
 
+You can check `skore`'s latest version on
+[PyPI](https://pypi.org/project/skore/).
+
 ## 🚀 Quick start
-=======
 
-In your shell, run the following to create a project file `project.skore` (the default) in your current working directory:
+1. From your shell, initialize a `skore` project, here named `project.skore`, that will be in your current working directory:
 ```bash
-python -m skore create 'project.skore'
+python -m skore create "project.skore"
 ```
-
-Run the following in your Python code (in the same working directory) to load the project, store some objects, delete them, etc:
+2. Then, from your Python code (in the same directory), load the project and store an integer for example:
 ```python
 from skore import load
-
-# load the project
 project = load("project.skore")
-
-# save an item you need to track in your project
 project.put("my int", 3)
+```
+3. Finally, from your shell (in the same directory), start the UI locally:
+```bash
+python -m skore launch project.skore
```

Comment:
Use of double-quotes is not consistent with previous call to `python -m skore create "project.skore"`.

Reply:
I don't understand, is it still the case now?

Reply:
Look at line 34, there are double quotes. And here no double quotes.

@thomass-dev shall we use double-quotes so that the documentation is uniform amongst all platforms?

Reply:
I added double quotes

Reply:
applied double quotes